harness/determined/cli/user_groups.py

```diff
@@ -0,0 +1,229 @@
+import json
+from argparse import Namespace
+from collections import namedtuple
+from typing import Any, Dict, List, Optional
+
+from determined.cli.experiment import pagination_args
+from determined.cli.session import setup_session
+from determined.common import api
+from determined.common.api import authentication, bindings
+from determined.common.declarative_argparse import Arg, Cmd
+from determined.common.experimental import session
+
+from . import render
+
+v1UserHeaders = namedtuple(
+    "v1UserHeaders",
+    ["id", "username", "displayName", "admin", "active", "agentUserGroup", "modifiedAt"],
+)
+
+v1GroupHeaders = namedtuple(
+    "v1GroupHeaders",
+    ["groupId", "name"],
+)
+
+
+@authentication.required
+def create_group(args: Namespace) -> None:
```

Comment:
I think we should check if group name is empty here

Reply:
Since group name is a positional arg here it can't be empty right?

Reply:
I'm not actually super sure about positional arg behavior, but it looks like `None` is provided as a default arg value, which would cause an error further down because the name field can't be null?